src/components/header.js

```diff
@@ -1,23 +1,112 @@
-import { Link } from "gatsby"
+import { Link as GatsbyLink } from "gatsby"
 import PropTypes from "prop-types"
 import React from "react"
-import { Box, Text } from "rebass"
+import { Box, Image, Flex, Link } from "rebass"
+import Logo from "../images/logo.svg"
+import Container from "./container"
 
 const Header = ({ siteTitle }) => (
-  <Box as="header" mb={2} bg="darkBlue">
-    <Box m="0 auto" maxWidth={960} px={1} py={2}>
-      <Text as="h1" fontSize={5} variant="heading">
-        <Link
+  <Box mb={0} as="header">
+    <Container>
+      <Flex color="white" alignItems="center" py={0}>
+        <GatsbyLink
           to="/"
           style={{
             color: `white`,
             textDecoration: `none`,
           }}
-        >
-          {siteTitle}
-        </Link>
-      </Text>
-    </Box>
+        />
+        <Image
+          src={Logo}
+          alt={siteTitle}
+          sx={{
+            width: ["206px", "50%"],
+          }}
+        />
+        <Box mx="auto" />
+        <Box>
+          <Link
+            mx={2}
+            p={0}
+            variant="nav"
+            href="/meetups"
+            sx={{
+              display: "inline-block",
+              transition: "400ms ease all",
+              textTransform: "uppercase",
+              "&:hover": {
+                color: "gold",
+              },
+              "&::after": {
+                content: '""',
+                display: "block",
+                width: "0",
+                height: "2px",
+                background: "gold",
+                transition: "width 300ms",
+              },
+              "&:hover::after": {
+                width: "100%",
+              },
+            }}
```

Comment:
Instead of repeating this, you can add this to the `theme.js`. Check [this](https://github.com/rebassjs/rebass/blob/b32d3958e227f57533fb0767dc2899e2ef3f9564/packages/preset/src/index.js#L59) as guide.

Declaring variants
```
variants: {
  nav: {
    ...styles
  }
}
```

The component with prop `variant="nav"` will inherit those styles.

Reply:
got it thanks didn't know this also possible with rebase. already fixed